backend/tarsy/agents/iteration_controllers/react_final_analysis_controller.py

```
@@ -64,10 +67,37 @@ async def execute_analysis_loop(self, context: 'StageContext') -> str:
         
         # Generate response and get the latest assistant message content
-        updated_conversation = await self.llm_client.generate_response(conversation, context.session_id, context.agent.get_current_stage_execution_id())
-        latest_message = updated_conversation.get_latest_assistant_message()
-        
-        if latest_message:
-            return latest_message.content
-        else:
-            return "No response generated from LLM"
\ No newline at end of file
+        try:
+            updated_conversation = await self.llm_client.generate_response(conversation, context.session_id, context.agent.get_current_stage_execution_id())
```

Ruff (0.12.2)

Line too long (156 > 88)

(E501)